tests/Aspire.Hosting.Azure.Tests/AzureContainerAppsTests.cs

```diff
@@ -236,9 +227,9 @@ public async Task AddContainerAppsInfrastructureAddsDeploymentTargetWithContaine
           "path": "api.module.bicep",
           "params": {
             "api_containerport": "{api.containerPort}",
-            "outputs_azure_container_registry_managed_identity_id": "{.outputs.AZURE_CONTAINER_REGISTRY_MANAGED_IDENTITY_ID}",
             "outputs_azure_container_apps_environment_id": "{.outputs.AZURE_CONTAINER_APPS_ENVIRONMENT_ID}",
             "outputs_azure_container_registry_endpoint": "{.outputs.AZURE_CONTAINER_REGISTRY_ENDPOINT}",
+            "outputs_azure_container_registry_managed_identity_id": "{.outputs.AZURE_CONTAINER_REGISTRY_MANAGED_IDENTITY_ID}",
```

Comment:
How is this change related to this PR? was this regeneration from an earlier PR?

Reply:
It moved around